cmake/atomic_configure_file.cmake

```diff
@@ -1,9 +1,11 @@
 function(atomic_configure_file input output)
-  set(atomic_file "${CMAKE_BINARY_DIR}/atomic_configure_file")
+  get_filename_component(atomic_file ${output} NAME)
+  set(atomic_file "${CMAKE_BINARY_DIR}/${atomic_file}")
   configure_file("${input}" "${atomic_file}" ${ARGN})
   get_filename_component(output_path ${output} PATH)
-  if(NOT EXISTS ${output_path})
-    file(MAKE_DIRECTORY ${output_path})
+  # sync multiple catkin cmake processes writing to that location
+  if (CMAKE_VERSION VERSION_GREATER 3.2)
```

Comment:
The condition evaluates to `FALSE` for CMake 3.2 for which it should be `TRUE`.

Reply:
Fixed.